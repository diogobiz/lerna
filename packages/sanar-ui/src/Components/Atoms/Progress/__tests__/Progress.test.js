
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESProgress from '../'

it('renders correctly', () => {
    const component = (
        <ESProgress>
            <h1>ESProgress</h1>
        </ESProgress>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
