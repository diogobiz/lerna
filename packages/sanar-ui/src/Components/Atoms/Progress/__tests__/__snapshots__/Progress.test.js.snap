// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-progress"
>
  <div
    className="ant-progress ant-progress-line ant-progress-status-normal ant-progress-default"
  >
    <div>
      <div
        className="ant-progress-outer"
      >
        <div
          className="ant-progress-inner"
        >
          <div
            className="ant-progress-bg"
            style={
              Object {
                "background": undefined,
                "borderRadius": "",
                "height": 8,
                "width": "0%",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
